scripts/topic_data_migration.py

```diff
@@ -0,0 +1,43 @@
+import django
```

Comment:
We don't add one-time scripts to the repo anymore. This script should be removed and instead manually copied to the server that will run it.

Reply:
Got it! Thanks @nsantacruz :)